src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,50 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.HashMap;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        HashMap<String, Integer> data = new HashMap<>();
+        try (BufferedReader br = new BufferedReader(new FileReader(fromFileName))) {
+            String value = br.readLine();
+            while (value != null) {
+                String[] str = value.split(",");
+                if (!data.containsKey(str[0])) {
+                    data.put(str[0], Integer.parseInt(str[1]));
+                } else {
+                    data.put(str[0], data.get(str[0]) + Integer.parseInt(str[1]));
+                }
+                value = br.readLine();
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read from file", e);
+        }
+        data.put("result", data.get("supply") - data.get("buy"));
+        writeStatistic(toFileName, createStatistic(data));
+    }
+
+    private String createStatistic(HashMap<String, Integer> data) {
```

Comment:
This method should accept the string with content of a file, then:
1. Split entries by line separator
2. Split each line into values
3. Parse values and process operation
4. Return results